namespace cpp cc.service.java
namespace java cc.service.java

service JavaService
{
  string getJavaString()
}